modules/serverless-kafka-broker.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-kafka-developer.adoc
+// * serverless/develop/serverless-using-brokers.adoc
+
+[id="serverless-kafka-broker_{context}"]
+= Creating a Kafka broker
```

Comment:
I believe we can also have use a "Creating a Kafka broker by using the Knative CLI" ?

```shell
$ kn broker create mybroker --class Kafka
```

Reply:
I'd vote for this being done in a separate PR.

Reply:
Agreed re separate PR. I think this can be documented as part of https://issues.redhat.com/browse/SRVKE-528 (I've update the Jira to reflect this).